sdk/py/apepay/settings.py

```diff
@@ -8,6 +8,7 @@
 class Settings(BaseSettings):
     WARNING_LEVEL: timedelta = timedelta(days=2)
     CRITICAL_LEVEL: timedelta = timedelta(hours=12)
+    MAX_STREAM_DURATION: int = int(timedelta.max.total_seconds()) - 1
```

Comment:
I don't think this needs to be a setting, would just make it a constant in the other file

Reply:
My thinking was that someone may want to have a max life, but this probably wouldn't be how we'd want to enforce that anyway.  Will change

Reply:
it's sort of a programming language issue

honestly, thinking it should just raise an error for being out of bounds

Reply:
Changed.  Let me know if you'd prefer me to throw an error.  If we did that it would still completely break StreamManager, then.

Reply:
how do you mean? I think it would make it break on a much clearer error, something like "stream duration out of bounds"

Reply:
I'm picturing this raising a bug on handling that the stream was improperly created, and should not be handled any further

Reply:
For instance, iteration of streams via `StreamManager` raises an exception from `time_left()`.  Meaning it's impossible to iterate through streams if one has a bad duration.

Reply:
If we just added a different exception, we'd have to handle it (and suppress the stream) in every iterator func by excluding the stream.  Suppression probably isn't the way to go, considering these aren't invalid streams.  This is just a limitation of Python.